common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -108,4 +133,38 @@ private void updateTextRenderSetting() {
     protected TextRenderSetting getTextRenderSetting() {
         return textRenderSetting;
     }
+
+    private Stream<StatusEffectWithProperties> stackEffects(List<StatusEffect> effects) {
```

Comment:
I don't think this method is fully correct. I might have the same effect, with different timers. Or, I might have the same effect, with different modifiers, and this doesn't take it into account, does it?

Reply:
`.asString()` returns the fullName from the StyledText class, which includes the modifier and timer.